engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlanConfiguration.java

```diff
@@ -112,14 +112,14 @@
   public static final String GATEWAY_PREFIX = StreamingApplication.DT_PREFIX + "gateway.";
   public static final String GATEWAY_LISTEN_ADDRESS = GATEWAY_PREFIX + "listenAddress";
 
-  public static final String STREAM_PREFIX = StreamingApplication.DT_PREFIX + "stream.";
+  public static final String STREAM_PREFIX = StreamingApplication.APEX_PREFIX + "stream.";
```

Comment:
Used by DumpPropertiesFileCommand. I don't think there is any user that depends on this functionality, but of course that is not a guarantee. However, since the properties file with the new key prefix would work again as input, I don't see a reason to spend time on implementing what you suggest. I would like another opinion on this, maybe @davidyan74 since he knows this code best?